lib/rdoc/options.rb

```diff
@@ -1017,17 +1008,6 @@ def parse argv
         @static_path << value
       end
 
-      opt.separator nil
-
-      opt.on("--webcvs=URL", "-W",
```

Comment:
We should keep this for https://github.com/ruby/rdoc/pull/1122#discussion_r1649806944

Reply:
@hsbt It has been restored. I believe `attr_accessor` part should remain removed since `RDoc::Option` class is not meant to be used directly by other gems.

Reply:
Can you also update this description with deprecated warning?

And we carefully to remove these code with https://github.com/ruby/rdoc/pull/1122#pullrequestreview-2139737789